context/app/static/js/components/publications/PublicationVisualizationsSection/PublicationsVisualizationSection.jsx

```diff
@@ -1,45 +1,54 @@
 import PrimaryColorAccordionSummary from 'js/shared-styles/accordions/PrimaryColorAccordionSummary';
+import SectionHeader from 'js/shared-styles/sections/SectionHeader';
 import { StyledAccordionDetails } from './style';
 
 function PublicationsVisualizationSection({ vignette_json: { vignettes }, uuid }) {
   const [expandedIndex, setExpandedIndex] = useState(0);
 
-  const handleChange = (i) => (event, isExpanded) => {
-    setExpandedIndex(isExpanded ? i : false);
-  };
+  const [displayedVignettes, setDisplayedVignettes] = useState({
+    ...[true].concat(Array(vignettes.length - 1).fill(false)),
+  });
+
+  const sortedVignettes = useMemo(() => {
+    return vignettes.sort((a, b) => a.directory_name.localeCompare(b.directory_name));
+  }, [vignettes]);
+
+  const handleChange = useCallback((i) => (event, isExpanded) => setExpandedIndex(isExpanded ? i : false), []);
 
   return (
     <DetailPageSection id="visualizations">
       <SectionHeader>Visualizations</SectionHeader>
-      {vignettes
-        .sort((a, b) => a.directory_name.localeCompare(b.directory_name))
-        .map((vignette, i) => {
-          return (
-            <Accordion
-              key={vignette.name}
-              expanded={i === expandedIndex}
-              TransitionProps={{ mountOnEnter: i !== 0 }}
-              onChange={handleChange(i)}
-            >
-              <PrimaryColorAccordionSummary $isExpanded={i === expandedIndex} expandIcon={<ArrowDropUpRoundedIcon />}>
-                <Typography variant="subtitle1">{`Vignette ${i + 1}: ${vignette.name}`}</Typography>
-              </PrimaryColorAccordionSummary>
-              <StyledAccordionDetails>
-                <PublicationVignette vignette={vignette} uuid={uuid} vignetteDirName={vignette.directory_name} />
-              </StyledAccordionDetails>
-            </Accordion>
-          );
-        })}
+      {sortedVignettes.map((vignette, i) => {
+        return (
+          <Accordion
+            key={vignette.name}
+            expanded={i === expandedIndex}
+            TransitionProps={{ onEntered: () => setDisplayedVignettes((prev) => ({ ...prev, [i]: true })) }}
+            onChange={handleChange(i)}
+          >
+            <PrimaryColorAccordionSummary $isExpanded={i === expandedIndex} expandIcon={<ArrowDropUpRoundedIcon />}>
+              <Typography variant="subtitle1">{`Vignette ${i + 1}: ${vignette.name}`}</Typography>
+            </PrimaryColorAccordionSummary>
+            <StyledAccordionDetails>
+              <PublicationVignette
+                vignette={vignette}
+                uuid={uuid}
+                vignetteDirName={vignette.directory_name}
+                mounted={displayedVignettes[i]}
+              />
+            </StyledAccordionDetails>
+          </Accordion>
+        );
+      })}
     </DetailPageSection>
   );
 }
 
-export default PublicationsVisualizationSection;
+export default memo(PublicationsVisualizationSection);
```

Comment:
```suggestion
export default memo(PublicationsVisualizationSection);
```
How does this impact the render performance in this case?

Reply:
This was a relic of a previous attempt that I forgot to revert before committing 😅 dropping it now